contracts/base/TokenCollateralLoans.sol

```diff
@@ -21,17 +21,21 @@ pragma experimental ABIEncoderV2;
 import "./LoansBase.sol";
 
 // Interfaces
-import "../interfaces/TokenLoansInterface.sol";
+import "../interfaces/LoansInterface.sol";
 
 
-contract TokenLoans is TokenLoansInterface, LoansBase {
+contract TokenCollateralLoans is LoansInterface, LoansBase {
```

Comment:
I think the LoansInterface can be moved to LoansBase contract, and then move all the _emitXXXXEvent implementations to LoansBase. So, parent contracts (and LoansBase) can call them.